CSV_Trans_Col_To_Rec_.py

```diff
@@ -1,71 +1,71 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-# vim:fenc=utf-8
+# vim:fenc=utf-8 
 #! /usr/bin/env python
 # csvファイル読み込みライブラリをImport
 import os
 import csv
 
 def trance(I_files,O_files,begin,column,repeat):
-#+---------+---------+---------                
+#+---------+---------+---------                 
 # I_files :     入力ファイル名（Fullpath)
 # O_files :     出力ファイル名（Fullpath)
 # keys :        行に展開しない列項目数（粒度やキーとしての役割）
 # degin :       開始は0列になるので13列目なら12
 # column :      7列分を移動するなら7
 # repeat :      列→行変換を繰り返す回数
-#+---------+---------+---------                
-#       入力ファイルを文字コードutf8で開く
+#+---------+---------+---------                 
+#       入力ファイルを文字コードutf8で開く 
         f = open(I_files, 'r',encoding="utf-8_sig")
```

Comment:
Generally, it's better to use `with` when opening files.  
When you use with, you also don't need `f.close()`, because `with` statement will automatically close the file when finished.
```suggestion
        with open(I_files, 'r' ,encoding="utf-8_sig") as f:
```